src/pudl_archiver/depositors/__init__.py

```diff
@@ -3,4 +3,60 @@
 Such as Zenodo, Zenodo, and Zenodo.
 """
 
-from pudl_archiver.depositors.zenodo import ZenodoDepositor  # noqa: F401
+from dataclasses import dataclass
+
+import aiohttp
+
+from pudl_archiver.frictionless import DataPackage
+from pudl_archiver.utils import RunSettings
+
+from .depositor import (
+    DepositionAction,
+    DepositionChange,
```

Comment:
I decided to import them here so they're available to import elsewhere from `pudl_archiver.depositors` instead of `pudl_archiver.depositors.depositor`.